src/models/user_indicator_model.rb

```diff
@@ -2,11 +2,10 @@
 class UserIndicatorModel
   attr_reader :definition
 
-  def initialize
+  def initialize(name: 'child')
```

Comment:
Also here: I'd prefer whole filename

Reply:
I agree, I think I like a refactoring where we initialize the UserIndicatorModel with the filename, and optional weight parameters, and then it either uses the default weight from the filename, or the given one.
I just did not want to have weight and filename doubled to much. (cause it seems reduntant), but if we think of the weight in the filename as the default one, everything works